gsa/src/web/components/dialog/savedialog.js

```diff
@@ -146,14 +170,18 @@ const SaveDialog = ({
   initialHeight,
   minHeight,
   minWidth,
+  multiStep = false,
   title,
   values,
   width,
   onClose,
   onError,
   onErrorClose,
   onSave,
+  ...props
 }) => {
+  const {nextDisabled, onNext, onPrev, prevDisabled} = props;
```

Comment:
This is equivalent to just putting these props into the block, where you added the ...props. Please just extend the list above instead of making the detour with this line.